### PR TITLE
Added documentation for new permission OnlineMeetingArtifact.Read.All

### PR DESCRIPTION
**Pull request by frankpeng7**
Added documentation for new permission OnlineMeetingArtifact.Read.All

### CODE REVIEW

File: concepts/permissions-reference.md [hunk 1]
```diff
@@ -1357,13 +1357,15 @@ The *CreatedByApp* constraint associated with this permission indicates that the
...
 
 #### Application permissions
 
 |Permission    |Display String   |Description |Admin Consent Required |
 |:-----------------------------|:-----------------------------------------|:-----------------|:-----------------|
-|_OnlineMeetings.Read.All_|Read Online Meeting details from the app |Allows the app to read online meeting details in your organization without a signed-in user.|Yes|
-|_OnlineMeetings.ReadWrite.All_|Read Online Meeting details from the app|Allows an app to create, read online meetings without a signed-in user.|Yes|
+|_OnlineMeetings.Read.All_|Read Online Meeting details from the app |Allows the app to read Online Meeting details in your organization without a signed-in user.|Yes|
+|_OnlineMeetings.ReadWrite.All_|Read Online Meeting details from the app|Allows an app to create, read Online Meetings without a signed-in user.|Yes|
+|_OnlineMeetingArtifact.Read.All_|Read Online Meeting artifacts from the app |Allows the app to read Online Meeting artifacts in your organization, without a signed-in user.|Yes|
```
Comment (Jackson-Woods) [inline concepts/permissions-reference.md:1368]: @frankpeng7 The display strings seem off, can you double check? I see "Read online meeting details" / "Read and create online meetings" / "Read online meeting artifacts" for the application permissions in the portal.

Comment (frankpeng7) [reply concepts/permissions-reference.md:1368]: @Jackson-Woods these are legacy docs I have no context around. Are the strings required to match the portal?

**Review by Lauragra — APPROVED** [review]

File: concepts/permissions-reference.md (hunk 1 above)
Comment (frankpeng7) [reply concepts/permissions-reference.md:1368]: @Jackson-Woods feel free to open an issue or ping me offline if you still have concerns. Resolving for now.

**Comment by frankpeng7** [comment]
@Lauragra this one is ready to merge. Can you help with it? Thanks!

**Pull request merged**